### PR TITLE
Split seed info into separate file, set up webpack and React

### PR DESCRIPTION
**Pull request by alexsheehan**

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+const db = require('./index.js');
+
+mongoose.connect('mongodb://localhost/opentable');
+
+const cuisines = ['American', 'Italian', 'Steakhouse', 'Seafood', 'French', 'Indian', 'Mexican', 'Japanese', 'Chinese', 'German', 'Spanish', 'Fusion/Eclectic', 'Barbecue', 'Greek', 'Grill', 'Comfort Food', 'Turkish', 'International', 'Thai', 'Tex-Mex', 'Bistro', 'Bar/Lounge', 'Californian'];
```
Comment (ktseto) [inline database/seed.js]: consider breaking the array into multiple lines for readability

File: client/src/components/Stars.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const fullStar = <svg><path className="red" d={icons.star.full} /></svg>;
+  const emptyStar = <svg><path className="grey" d={icons.star.full} /></svg>;
+  const halfStar = (
+    <svg>
+      <path className="grey" d={icons.star.full} />
+      <path className="red" d={icons.star.half} />
+    </svg>
+  );
+  const allStars = [];
+  for (let i = 0; i < Math.floor(stars); i += 1) {
```
Comment (ktseto) [inline client/src/components/Stars.jsx:16]: I think eslint dislikes for-loops in general so I've been trying to use forEach or _.range where possible.  Not sure there's an easy replacement here but just a thought.

**Review by ktseto — COMMENTED** [review]
looks great! just a couple minor stylistic comments. great job!

File: client/src/components/Stars.jsx (hunk 2 above)
Comment (alexsheehan) [reply client/src/components/Stars.jsx:16]: My eslint isn't throwing an error for this, but I'll keep it in mind!

File: database/seed.js (hunk 1 above)
Comment (alexsheehan) [reply database/seed.js]: Done

**Review by alexsheehan — COMMENTED** [review]
Modified based on suggestions

**Pull request merged**